internal/cmd/connector_test.go

```diff
@@ -198,7 +200,19 @@ func TestConnector_Run_Kubernetes(t *testing.T) {
 	})
 	assert.DeepEqual(t, fakeKube.writes, expectedWrites, cmpKubeRequest)
 
-	// TODO: check proxy is listening
+	<-opts.Addrs.Available
+	u := fmt.Sprintf("https://%v/api/v1/namespaces/default/pods/podok", opts.Addrs.HTTPS.String())
```

Comment:
what's at this endpoint?

Reply:
Nothing, the service behind the proxy is a fake. We only care that the proxy is handling requests